.circleci/config.yml

```diff
@@ -64,13 +64,41 @@ jobs:
       - store_test_results:
           path: test-results
 
+  py3_compat:
+    docker:
+      - image: circleci/python:2.7
+    steps:
+      - checkout
+      - run:
+          name: install futurize
+          command: pip install future --user
+      - run:
+          name: find python3-incompatible code
+          command: |
+            FUTURIZE="futurize ."
+            while read line; do
+              [[ "$line" =~ ^#.* ]] && continue
+              FUTURIZE="${FUTURIZE} ${line}"
```

Comment:
note that this is different from the cloudify-common by this line, because in common we do `-f ${line}`, and here just `${line}` which means that the file contains flags, eg. `--stage1` or `-f specific.fixer`.

This was more useful here, because there's less changes, so it's easier to spell out `--stage1` than the 30 individual fixers that go into it.

This is probably a better way, and we should do that everywhere